Resources/Prototypes/Entities/Clothing/Shoes/magboots.yml

```diff
@@ -56,6 +56,22 @@
   - type: StealTarget
     stealGroup: ClothingShoesBootsMagAdv
 
+- type: entity
+  parent: ClothingShoesBootsMag
+  id: ClothingShoesBootsMagSci
+  name: magboots
+  description: Magnetic boots, often used during extravehicular activity to ensure the user remains safely attached to the vehicle.
+  components:
+  - type: Sprite
+    sprite: Clothing/Shoes/Boots/magboots-science.rsi
+    state: icon
```

Comment:
you probably should just remove the state definition here and just inherit the layers from the base one. Having both a state and layers defined is kinda weird and i'm not sure how it works.